Library/Homebrew/brew.sh

```diff
@@ -374,6 +376,19 @@ then
   export HOMEBREW_BOTTLE_DOMAIN="$HOMEBREW_BOTTLE_DEFAULT_DOMAIN"
 fi
 
+HOMEBREW_DEFAULT_BREW_REMOTE="https://github.com/Homebrew/brew.git"
+if [[ -z "$HOMEBREW_BREW_REMOTE" ]]
+then
+  HOMEBREW_BREW_REMOTE="$HOMEBREW_DEFAULT_BREW_REMOTE"
+fi
+
+if [[ -n "$HOMEBREW_MACOS" ]] || [[ -n "$HOMEBREW_FORCE_HOMEBREW_ON_LINUX" ]]
+then
+  HOMEBREW_DEFAULT_CORE_REMOTE="https://github.com/Homebrew/homebrew-core"
+else
+  HOMEBREW_DEFAULT_CORE_REMOTE="https://github.com/Homebrew/linuxbrew-core"
+fi
```

Comment:
Sorry, my mistake: all this logic will need moved above the `export HOMEBREW_BREW_REMOTE` above